scripts/join-release-testnet.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# Set up a Exocore service to join the Exocore Release testnet.
+
+# Load environment variables from .env file
+if [ -f .env ]; then
+  export $(cat .env | xargs)
+fi
+
+# Configuration
+# You can have these values set in .env file, and run 'source .env' before running this script.
+# ***
+NODE_HOME=${NODE_HOME:-$HOME/.exocored}
+NODE_MONIKER=${NODE_MONIKER:-release-testnet}
+STATE_SYNC=${STATE_SYNC:-true}
+STATE_SYNC=$(echo "$STATE_SYNC" | tr '[:upper:]' '[:lower:]')
+
+if [ "$STATE_SYNC" = "true" ]; then
+  STATE_SYNC=true
+else
+  STATE_SYNC=false
+fi
+
+# Exocore configuration
+# Default values for Exocore testnet
+SERVICE_NAME=exocore
+EXOCORE_VERSION=1.0.2
+CHAIN_BINARY_URL=https://github.com/ExocoreNetwork/exocore/releases/download/v$EXOCORE_VERSION/exocore_$EXOCORE_VERSION\_Linux_amd64.tar.gz
```

Comment:
For support on ARM and Darwin devices, I wrote this in the validator documentation.
```
VERSION="1.0.2"
wget -O exocored_${VERSION}.tar.gz "https://github.com/ExocoreNetwork/exocore/releases/download/v${VERSION}/exocore_${VERSION}_$(uname -s)_$(uname -m | sed 's/x86_64/amd64/' | sed 's/aarch64/arm64/').tar.gz"
```